docs/convergence.rst

```diff
@@ -33,6 +33,51 @@ Will give a plot looks like this
    A convergence plot of showing that the forward and backward has converged
    fully.
 
+Fraction Convergence
```

Comment:
@VOD555 what is the name for the method?

Reply:
It originated from Yang W, Bitetti-Putzer R, Karplus M's paper, "Free energy simulations: Use of reverse cumulative averaging to determine the equilibrated region and the time required for convergence". Maybe we can call our method forward & reverse cumulative averaging.

Reply:
We have already had one forward_backward_convergence
https://alchemlyb.readthedocs.io/en/latest/convergence/alchemlyb.convergence.convergence.html#alchemlyb.convergence.forward_backward_convergence

Maybe we can call this cumulative_averaging?

Reply:
How about "reverse cumulative averaging" — quite similar to the original paper?

We should then also mention the Yang paper. @VOD555 can you please make a suggestion on the PR how to change the docs?

Reply:
Then call it "reverse cumulative averaging". I think we should mention that the ` forward_backward_convergence` is done on all samplings for intermediate states, and the "reverse cumulative averaging" here is for a single trajectory.

Reply:
I'm not sure if `reverse cumulative averaging` is good as `forward cumulative averaging` is done as well. I think the `forward & reverse cumulative averaging` is good for description but is too long for the name.

Reply:
How about `fwdrev_cum_avg()` or `FR_cumulative_average()` for a function name? The method name itself could be 
*Forward/reverse cumulative averaging* as it doesn't really matter how many letters it takes up. It is then similar to the original Yang paper but also includes the "forward" component, which is important for the convergence condition.

Reply:
I like the name of fwdrev_cum_avg. For the function, we have `R_c` and `A_c`, if we rename the function `R_c` to `fwdrev_cum_avg`, what should we do with A_c?

Reply:
I re-read the documentation and looked at things in context. It reminded me that the *fractional* part is an important detail that makes everything comparable. I'd therefore suggest a section title **Fractional equilibration time** — that's pretty much what we called $R_c$ in the paper.

I still like `R_c` but I also understand that the function returns _both_ $R_c$ and the data. I'd call the current `R_c()` function  `fwdrev_cumavg_Rc()` (numpy has `cumsum()` so `cumavg` should look familiar, `fwdrev` nods to the original method, and `Rc` connects it to the fractional equilibration time) — a bit clunky but at least gives an idea that one gets multiple things back.

I'd keep `A_c()` as it makes it easy to relate it to the math and the paper.

Sorry for the back and forth... that's now my opinion and I am going to stand by it. Let me know what you think.

p.s.: The image should say "energy/kT" instead of $\Delta G/kT$.

Reply:
I think this is reasonable, will address the comments this weekend.